src/wp-includes/class-wpdb.php

```diff
@@ -1587,7 +1587,18 @@ public function print_error( $str = '' ) {
 			'error_str' => $str,
 		);
 
-		if ( $this->suppress_errors ) {
+		/**
+		 * Filters whether to suppress a query error.
+		 *
+		 * @since 6.2.0
+		 *
+		 * @param boolean $suppress_error Whether to suppress the error.
```

Comment:
```suggestion
		 * @param bool $suppress_error Whether to suppress the error.
```
Core uses `bool` rather than `boolean`.